platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/store/ManagedServiceConfigurationFileTest.java

```diff
@@ -44,6 +44,9 @@
     private ConfigurationAdmin mockConfigAdmin;
 
     @Mock
+    private PersistenceStrategy persistenceStrategy;
```

Comment:
Since the other fields start with `mock`, we should either prefix this one with `mock` or remove `mock` from all of them.